### PR TITLE
[Bugfix] Storybook command fails

### PR DESCRIPTION
**Pull request by jcalcaben**
## Description

Updates mostly Storybook configuration files to prevent the Storybook command from failing in the core project and scaffolded projects.

## Related Issue

Closes PWA-2230

## Acceptance

Any developer

### Verification Stakeholders

Any developer

## Verification Steps

1. Run storybook command: `yarn storybook:venia`
2. Verify storybook starts
3. Verify the Header story looks correct (i.e. tailwind styles have been applied)
4. Create a new scaffolded project:

```
DEBUG_PROJECT_CREATION=1 yarn buildpack create-project scaffold-test --template "packages/venia-concept" --name "scaffold-test" --author "Scaffold Test" --backend-url "https://master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud/" --npm-client "yarn"
```
5. Run storybook command in the scaffolded project: `yarn run storybook`
6. Verify storybook starts
7. Verify the Header story looks correct (i.e. tailwind styles have been applied)

#### Test scenario(s) for direct fix/feature

1. Verify storybook starts and Header test looks correct

#### Test scenario(s) for any existing impacted features/areas

1. Verify storybook still runs

#### Is Browser/Device testing needed?

No

## Checklist

-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.

### CODE REVIEW

File: postcss.config.js [hunk 1]
```diff
@@ -4,6 +4,6 @@
 module.exports = {
     plugins: [
         require('autoprefixer'),
-        require('tailwindcss')('./tailwind.config.js')
+        require('tailwindcss')('../venia-concept/tailwind.config.js')
```
Comment (JustinConabreeAbsolunet) [inline postcss.config.js:7]: @jcalcaben Just curious how this works. `../` would put it outside of the project no?

**Review by JustinConabreeAbsolunet — APPROVED** [review]
✅ LGTM!

File: postcss.config.js (hunk 1 above)
Comment (jcalcaben) [reply postcss.config.js:7]: I think it's a quirk of the monorepo setup. When we build or run storybook it uses the `postcss.config.js` at the root of the project, but the working directory is within that package (either `venia-concept` for development or `venia-ui` for storybook). At least that's my best guess to how it works.

Comment (JustinConabreeAbsolunet) [reply postcss.config.js:7]: Got it. Thanks! 😄

**Comment by dpatil-magento** [comment]
QA Approved

**Comment by lilbumblebear** [comment]
Quick question about this fix.  I work on multiple storefronts with different extensions and theme development. My project structure is: 

    pwa-studio
        packages
               extensions
                        extension1
                        extension2
               storefronts
                        storefront1
                        storefront2
               (all the normal pwa stuff)... venia-ui, venia-concept, etc.
        postcss.config.js

When the change in postcss.config.js is made, my watch command breaks for my storefronts.  It's not terrible to change the file if I need to see the storybook, but is there a preferred way to setup a project for multi-store and multi-extension development so this would not break?

**Pull request merged**